Source/Eto.Test/Eto.Test/Sections/Drawing/BrushSection.cs

```diff
@@ -312,6 +315,22 @@ Control UseBackgroundColorControl()
 			return control;
 		}
 
+		Control WithContent()
+		{
+			var control = new CheckBox() { Text = "With Content" };
+			control.CheckedChanged += (s, e) =>
+			{
+				drawable.Content = ( (CheckBox)s ).Checked == true ?
+				TableLayout.AutoSized(
+					new Label() {
+						Text = "Some text",
+						BackgroundColor = Colors.Transparent },
+					centered: true
+				).With(tl => tl.BackgroundColor = Color.FromArgb(0x40000000)) : null;
+			};
+			return control;
+		}
+
```

Comment:
This tests the transparency of content over drawable vs. brushes.
It can help to implement and test it on other platforms (e.g. D2D).